src/components/sync-connected-sites.tsx

```diff
@@ -301,28 +329,37 @@ const SyncConnectedSitesSection = ( {
 																detail,
 															} );
 														} }
-														disabled={ isAnySitePulling || isAnySitePushing || isOffline }
+														disabled={ isAnySiteSyncing || isOffline }
 													>
 														<Icon icon={ cloudDownload } />
 														{ __( 'Pull' ) }
 													</Button>
 												</Tooltip>
 												<Tooltip
-													text={ getLastSyncTimeWithType(
-														selectedSite.id,
-														connectedSite.id,
-														'push'
-													) }
+													text={
+														isAnySiteSyncing
+															? isAnyConnectedSiteSyncing
+																? __(
+																		'This Studio site is syncing. Please wait for the sync to finish before you push this site.'
```

Comment:
Similarly as [above](https://github.com/Automattic/studio/pull/718/files#r1872972403):

```suggestion
																		'This Studio site is syncing. Please wait for the sync to finish before you push it.'
```